language-server/modules/langserver-compiler/src/main/java/org/ballerinalang/langserver/compiler/common/modal/BallerinaFile.java

```diff
@@ -56,6 +61,11 @@ public void setBallerinaProject(boolean ballerinaProject) {
         isBallerinaProject = ballerinaProject;
     }
 
+    public BallerinaFile setCompilerContext(CompilerContext compilerContext) {
```

Comment:
Shall we make this method return `void`?

Reply:
This was mistakenly added. Will send another PR with the fix